src/Model/DataModel/StreamData/HoldEmailData.php

```diff
@@ -40,19 +45,48 @@ class HoldEmailData extends StreamData
     public $deliveryLocation = '';
 
     /**
+     * @var bool
+     */
+    public $success;
+
+    /**
      * @param Patron $patron
      * @param Bib $bib
      * @param Item $item
      * @param HoldRequest $holdRequest
+     * @param HoldRequestResult $holdRequestResult
      */
-    public function assembleData(Patron $patron, Bib $bib, Item $item, HoldRequest $holdRequest)
+    public function assembleData(Patron $patron, Bib $bib, Item $item, HoldRequest $holdRequest, HoldRequestResult $holdRequestResult)
     {
         $this->setAuthor($bib->getAuthor());
         $this->setBarcode($item->getBarcode());
-        $this->setPatronName($patron->getNames()[0]);
         $this->setTitle($bib->getTitle());
         $this->setDeliveryLocation($holdRequest->getDeliveryLocation());
         $this->setPickupLocation($holdRequest->getPickupLocation());
+        $this->setSuccess($holdRequestResult->isSuccess());
+
+        $name = $patron->getNames()[0];
+        $fullName = explode(",", $name);
+        $fullName[1] = trim($fullName[1]);
+        $name = ucfirst(strtolower($fullName[1])) . " " . ucfirst(strtolower($fullName[0]));
+        $this->setPatronName($name);
```

Comment:
I was thinking about it as a display problem. It may make more sense to format the Patron's name in the Patron data class.